### DISCUSSION TITLE
next/future/image - Style property values shouldn't contain a semicolon

### DISCUSSION THREAD

**Discussion by Jahrhause** [vercel/next.js#38404] [Help] [answered]
Hi 

Im using the new experimental Image fro Next.js.
When i use the blurDataURL with a simple color baseURL i get the follwing warning/error:

`Warning: Style property values shouldn't contain a semicolon. Try "backgroundImage: url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http%3A//www.w3.org/2000/svg' xmlns%3Axlink='http%3A//www.w3.org/1999/xlink' viewBox='0 0 1200 600'%3E%3Cfilter id='b' color-interpolation-filters='sRGB'%3E%3CfeGaussianBlur stdDeviation='50'%3E%3C/feGaussianBlur%3E%3CfeComponentTransfer%3E%3CfeFuncA type='discrete' tableValues='1 1'%3E%3C/feFuncA%3E%3C/feComponentTransfer%3E%3C/filter%3E%3Cimage filter='url(%23b)' x='0' y='0' height='100%25' width='100%25' href='data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAADUlEQVR42mOsuPSgHgAGnwKrOCVWuwAAAABJRU5ErkJggg=='%3E%3C/image%3E%3C/svg%3E")" instead.`

The way I call it is:

`<Image
      src={src}
      alt=""
      width={1200}
      height={600}
      placeholder="blur"
      blurDataURL={
        'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAADUlEQVR42mOsuPSgHgAGnwKrOCVWuwAAAABJRU5ErkJggg=='
      }
      priority
    />`

What could I be doing wrong?

**Comment by Jahrhause** [comment]
Hi @valcosmos 

Thanx for your answer. Are you using the new next/future/image also?

This is a screen of my code:
![image](https://user-images.githubusercontent.com/58724862/177736638-b1e30537-1ffb-48cb-933d-c3286556dbb1.png)

I only get the error the first time, when I start the server (npm run dev) all other subsequent calls to the image runs fine.
![image](https://user-images.githubusercontent.com/58724862/177737591-10b6e8da-1c8d-47d5-b943-1cc9132fb056.png)

**Comment by icyJoseph** [comment, marked as answer]
Hi,

This warning is raised by React, and it is caused on this line https://github.com/vercel/next.js/blob/3411794607bc0a6d850a4567fbcea43bcd093f3c/packages/next/client/future/image.tsx#L510

```js
const imgStyle = Object.assign({}, style)
const svgBlurPlaceholder = `url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http%3A//www.w3.org/2000/svg' viewBox='0 0 ${widthInt} ${heightInt}'%3E%3Cfilter id='b' color-interpolation-filters='sRGB'%3E%3CfeGaussianBlur stdDeviation='50'/%3E%3CfeComponentTransfer%3E%3CfeFuncA type='discrete' tableValues='1 1'/%3E%3C/feComponentTransfer%3E%3C/filter%3E%3Cimage filter='url(%23b)' x='0' y='0' height='100%25' width='100%25' href='${blurDataURL}'/%3E%3C/svg%3E");`
```

React doesn't really like when the styles end with `;` like:

```jsx

<p
  style={{
    color: "red;",
  }}
>
  hello
</p>
```

You can try it on a codesandbox.

I guess this has to be fixed by the `next` team in the `next/future/image`.

**Reply by Jahrhause** [reply]
Thanks I'll raise an issue on this :)

**Reply by styfle** [reply]
PR with the fix #38454
